### DISCUSSION TITLE
I need a community, and those who can hold my hand.

### DISCUSSION THREAD

**Discussion by laurent-x1** [Cyfrin/foundry-full-course-cu#1642] [General]
Hello everyone and anyone who sees this, I am Laurent. I am completely new and would appreciate all the help i can get breaking into this world as soon as possible. I am ready to give in the time.
Looking forward to appreciating all the responses.
link me up with mentors, communities and projects.

**Comment by EngrPips** [comment]
Hey, @laurent-x1, Welcome aboard. I must mention that this space is super helpful as people are always willing and ready to help, so whenever you run into a problem while taking the course associated with this Github, you can leverage this discussion to ask questions and clarify your confusion.

**Reply by laurent-x1** [reply]
Thank you for the warm welcome @EngrPips, I will post my challenges whenever i experience them.
grateful for the reply.

**Reply by EngrPips** [reply]
You are welcome friend and I am looking forward to those.

**Comment by AbigirlMuchineripi** [comment]
"Welcome, to the ultimate Solidity, Smart Contract, Web3 blockchain development course, foundry edition...", in Patrick's voice lol. Welcome once again @laurent-x1 ,  there are great people here to help you. The community is here and you are part of it.

**Reply by laurent-x1** [reply]
Thanks Abi

**Comment by debsouryadatta** [comment]
Yea, even I am new here, lets grow and learn together!💪

**Reply by EngrPips** [reply]
Welcome aboard, friend. I wish you fast growth.

**Reply by laurent-x1** [reply]
Thank you debsoury

**Reply by debsouryadatta** [reply]
> Welcome aboard, friend. I wish you fast growth.

Yeah Thanks 🚀

**Reply by EngrPips** [reply]
You are welcome. Friend.

**Reply by Lytlotlo** [reply]
I'm new here too! Wish you all the best

**Comment by pate87** [comment]
Hey have a look at LearnWeb3.io and Discord as well as to Alchemy they have free workshops / hackathons around web 3 dev

**Reply by EngrPips** [reply]
Interesting suggestion.
